trace-gradle-plugin/src/functionalTest/java/io/bitrise/trace/plugin/util/FunctionalTestWriter.java

```diff
@@ -106,7 +91,7 @@ public void write(@NonNull final char[] chars, final int i, final int i1) {
 
     /**
      * Inserts to the start the coloring chars to a given array of chars. Please note that adding the coloring will
-     * stay until changed. To reset it to the default please use the {@link #reset()} method.
+     * stay until changed.
```

Comment:
nit: not sure this javadoc makes sense "the coloring will stay until changed" - i think the "until changed" isn't helpful - how do we change it? Perhaps something like "until a new color is requested, or the functional tests complete"?

Reply:
Thanks for spotting it, removed them.